sqlcapture/main.go

```diff
@@ -17,7 +18,7 @@ import (
 
 // Resource represents the capture configuration of a single table.
 type Resource struct {
-	Mode BackfillMode `json:"mode" jsonschema:"title=Backfill Mode,description=How the preexisting contents of the table should be backfilled. This should generally not be changed.,enum=Normal,enum=Only Changes,enum=Without Primary Key,default=Normal"`
+	Mode BackfillMode `json:"mode,omitempty" jsonschema:"title=Backfill Mode,description=How the preexisting contents of the table should be backfilled. This should generally not be changed.,default=,enum=,enum=Normal,enum=Precise,enum=Only Changes,enum=Without Primary Key"`
```

Comment:
This is a _little_ odd, but I see how you're using Go's default zero behavior here to avoid caring whether there's an explicit `""` or it's just not set.

Optional nit: I believe you could instead have this be `*BackfillMode` and then remove the `""` variant.
It's probably not worth changing, though, given how it impacts the switch statements that use Mode.

Reply:
Yeah, I went back and forth a couple of times on how exactly to represent this property's optionality and default value. I think the current approach is the least ugly overall, but I'd be happy to change it if you'd prefer.

Reply:
Nope, good as-is, especially if you've already considered it.